test/visual.test.js

```diff
@@ -0,0 +1,82 @@
+import Promise from 'bluebird';
+import test from 'tape';
```

Comment:
How come did you decide to go with `tape`? TAP output?
Introducing a new framework just this small file seems an overkill to me.